test/integration/dialects/postgres/dao.test.js

```diff
@@ -332,6 +332,31 @@ if (dialect.match(/^postgres/)) {
         });
       });
 
+      it('should not rename hstore fields', function() {
```

Comment:
No. That was where I tried put it before I knew where the issue was. Simply put, since the problem is caused by the DOA code, It cannot be a unit test.